src/celeritas/ext/detail/GeantGeoConverter.cc

```diff
@@ -707,21 +716,20 @@ VUnplacedVolume* GeantGeoConverter::convert(G4VSolid const* shape)
     else if (auto refl = dynamic_cast<G4ReflectedSolid const*>(shape))
     {
         G4VSolid* underlyingSolid = refl->GetConstituentMovedSolid();
-        CELER_LOG(debug)
-            << " Reflected solid found: "
-            << " volume: " << refl->GetName()
-            << " type = " << refl->GetEntityType()
-            << "   -- underlying solid: " << underlyingSolid->GetName()
-            << " type = " << underlyingSolid->GetEntityType();
+        CELER_ASSERT(underlyingSolid);
+        CELER_LOG(error) << "Encountered unsupported reflected solid '"
```

Comment:
So, if this error happens, why don't we assert here when CUDA=ON as the underlyingSolid always will trigger an exception when copied to GPU?